src/Compilers/Test/Core/Microsoft.CodeAnalysis.Test.Utilities.csproj

```diff
@@ -105,6 +105,8 @@
     <PackageReference Include="Basic.Reference.Assemblies.NetStandard20" Version="$(BasicReferenceAssembliesNetStandard20Version)" />
     <PackageReference Include="Basic.Reference.Assemblies.Net50" Version="$(BasicReferenceAssembliesNet50Version)" />
     <PackageReference Include="Basic.Reference.Assemblies.Net60" Version="$(BasicReferenceAssembliesNet60Version)" />
+
+    <PackageReference Include="Microsoft.ILVerification" Version="6.0.0" Condition="'$(TargetFramework)' == 'net472'" />
```

Comment:
Why are we only referencing this for `net472`? Would expect ILVerify to work on all target frameworks.

Reply:
I thought I'd reverted that. I was trying to solve the problem with strong name, assuming that the problem was only occurring for net472.

Reply:
Have we followed up with the team yet about why it's not strong named? That is our existing recommendation so it's odd that it is not being followed. I'm happy to chat with the owners about this.